tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -62,3 +69,61 @@ def test_models(
         name_0="hf",
         name_1="vllm",
     )
+
+
+@pytest.mark.parametrize("model", FP8_MODELS)
+@pytest.mark.parametrize("kv_cache_dtype", ["fp8"])
```

Comment:
I feel you could just add this parameter to the above test so that you don't have to duplicate an entire test function.

Reply:
I tried this but as you notice, I'm not using hf to validate but a completely different way (validate against the non chunked prefill version and only the log probs, not the generation)

Reply:
The reason is because of the Numerical error due to fp8 quant

Reply:
You can still merge them to one test and just use different checks for different types?
(Also please comment on the test for reasons.

Reply:
Sure but it's kind of swiss cheesing at that point as all of the arguments and output variables are different

If you want I can name this test `test_models_log_probs` `while the other is test_models_exact_match`

Reply:
Ok how about we keep this test but
1. rename to something like `test_fp8_model_and_kv_cache`, and
2. add docstring to explain why we compare logprobs.

Also for `facebook/opt-125m`, since you're using FP16 checkpoint which doesn't have kv cache scaling factors, you should test e5m2 for better accuracy.

Reply:
Ok sounds good, maybe that was the reason for the illegal memory access issue too let me check.

Reply:
Unfortunately; still illegal memory access

Reply:
Unfortunately, still illegal memory access